### ISSUE TITLE
Tab character in subject header shows as whitespace

### ISSUE THREAD

**Issue by romanofski** [purebred-mua/purebred-email#13]
I've noticed this in an e-mail today where the subject was something like:
```
  cli can'    t be used for
```
because the e-mail underneath looked like this:

```
Subject: [Bug 1562010] New: rpmdeplint cli can'                                                                                                                                                                                               
_¸t be used for checking repoclosure of a single repo
```
Obviously I would want it to read:
```
[Bug 1562010] rpmdeplint cli can't be used for checking repoclosure of a single repo
```
without the whitespace.
Have not spent time figuring out where exactly the problem is or what it supposed to happen, but rather thought filing the bug to have a record of this.

**Comment by frasertweedale** [comment]
@romanofski can you provide minimum reproducer for *purebred-email* itself.  AFAICT it's doing the right thing.

**Comment by romanofski** [comment]
I can try. Hm.. although when you say a single space then it seems to be doing the right thing... it just looks odd.

Checking in another mailer - mutt - it parses the mail the same. So... maybe there is no bug, it's just whatever produced the mail (guess Bugzilla) just stuffed up the word?

**Comment by frasertweedale** [comment]
Yeah, no bug :)  Closing.

**Comment by romanofski** [comment]
Today I realised that the problem was actually never purebred-email. Working on the serialisation made me realise that our subject headers are still not correctly "unfolded" field bodies. Looking into this closer, I realised that we're actually showing the subject out of the index from notmuch. So it's notmuch which seems to not correctly unfolding field bodies. Once you view the mail, a folded field body (e.g the subject) is correctly shown.
